ocs_ci/ocs/cluster.py

```diff
@@ -1362,6 +1362,58 @@ def check_ceph_osd_tree_after_node_replacement():
     return True
 
 
+def silence_ceph_osd_crash_warning(osd_pod_name):
+    """
+    Silence the osd crash warning of a specific osd pod
+
+    Args:
+        osd_pod_name (str): The name of the osd pod which we need to
+        silence the crash warning
```

Comment:
Please add one more indentation:

```python
"""
        osd_pod_name (str): The name of the osd pod which we need to
            silence the crash warning
"""

Reply:
Are you sure about that?

Reply:
yes, see here https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html
![Screenshot from 2020-11-29 16-31-18](https://user-images.githubusercontent.com/6973523/100544731-5d63c180-3260-11eb-8acf-f150da77224d.png)

Reply:
yes I see